src/LineItem/Command/FindLineItemByKeyCommand.php

```diff
@@ -45,6 +45,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
 		foreach ($lineItems as $lineItem) {
 			printf(" - order: %s, line item: %s\n", $lineItem['order_id'], $lineItem['line_item_id']);
 		}
+
+		if (empty($lineItems)) {
```

Comment:
I think it can be moved to line 44, and "early return" can be done after `printf( "No line-items found for given keys" )`